<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicalfileWindow</class>
 <widget class="QMainWindow" name="MedicalfileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{
 background :  ; 
 border-radius : 60px ; 
}
QLabel
{
 color : #48494B  ; 
}
QLineEdit
{
 background:transparent ; 
border : none ;
color:#48494B ;
font-size : 20px ; 

}
QTabWidget::pane {
  top:-40px; 
} 
QTabBar::tab {
background-color: rgb(217, 221, 220);
border-radius : 15px;
border: 2px solid lightgray; 
border-color: rgb(255, 255, 255);
padding: 3px;
} 

QTabBar::tab:selected { 
background-color: rgb(217, 221, 220);
border: 3px solid lightgray; 
border-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 140, 0);
border-right-color: rgb(255, 140, 0);
margin-bottom: 0px; 
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_update">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>1121</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(217, 221, 220);
border-radius : 35px ; </string>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_MedicalFile">
     <attribute name="title">
      <string>Create Medical File</string>
     </attribute>
     <widget class="QLabel" name="label_medical_file">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Medical File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_logo">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_condition">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>140</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-bottom:0.5px solid #717072 ; </string>
      </property>
      <property name="placeholderText">
       <string>Condition</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_description">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Description : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_next_meeting">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>520</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Next Meeting Date :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_save">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>500</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_description">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>250</y>
        <width>371</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_last_consultation">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>480</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 12pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);
border-radius : 15px ;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_last_consultation">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>470</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Last Consultation Date : </string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_next_meeting">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>530</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 12pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);
border-radius : 15px ;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_condition">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>150</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/terms-and-conditions.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>id_dossier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_MF_Managment">
     <attribute name="title">
      <string>Medical File Managment</string>
     </attribute>
     <widget class="QLabel" name="label_list">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>501</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>List Medical File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>230</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-bottom:0.5px solid #717072 ; </string>
      </property>
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_search">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>230</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/search.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_statistics">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Statistics</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>590</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>590</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>update</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_list">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>291</y>
        <width>931</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>571</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_calendrier">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>210</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>calendrier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Statistics">
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QLabel" name="label_list_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>681</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Statistics of Medical File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_logo_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>170</y>
        <width>611</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>update</string>
     </attribute>
     <widget class="QLabel" name="label_logo_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>520</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>update</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>id_dossier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_condition_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>170</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/terms-and-conditions.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_condition_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>160</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-bottom:0.5px solid #717072 ; </string>
      </property>
      <property name="placeholderText">
       <string>Condition</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_description_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>230</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Description : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_description_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>270</y>
        <width>371</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_last_consultation_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>500</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Last Consultation Date : </string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_last_consultation_2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>500</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 12pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);
border-radius : 15px ;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_next_meeting_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>580</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Next Meeting Date :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_next_meeting_2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>580</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 12pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);
border-radius : 15px ;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_list_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>551</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Update Medical File</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>calendrier</string>
     </attribute>
     <widget class="QLabel" name="label_medical_file_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>381</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>rendez_vous</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>411</width>
        <height>218</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_logo_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1195</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
